internal/action/lidarr.go

```diff
@@ -43,19 +43,28 @@ func (s *service) lidarr(ctx context.Context, action *domain.Action, release dom
 		cfg.Password = client.Settings.Basic.Password
 	}
 
-	arr := lidarr.New(cfg)
+	externalId := 0
+	if client.Settings.ExternalDownloadClientId > 0 {
```

Comment:
This is backwards. This is saying the parent takes priority over the wellbeing of the child 😄. The outcome is if this is set at the global level, you'll need another id set to override at the time of execution with one that isn't set. If it's inverted, it can still be overridden in the action.